### PR TITLE
correctly take unstable-features

### LINKED ISSUE #127: -Z unstable flags not passed to build command
executed command:
```
cargo size --release --verbose -Z unstable-options
```
verbose output:
```
"/usr/local/rustup/toolchains/nightly-2022-06-15-x86_64-unknown-linux-gnu/bin/cargo" "build" "--release" "--message-format=json"
error: the `--out-dir` flag is unstable, pass `-Z unstable-options` to enable it
See https://github.com/rust-lang/cargo/issues/6790 for more information about the `--out-dir` flag.
error: Failed to parse crate metadata
```
binutils version:
```
0.3.6
```

### PR DESCRIPTION
**Pull request by Emilgardis**
fixes #127

### CODE REVIEW

**Comment by Emilgardis** [comment]
ci errors are one of the issues with not having a lockfile :), we could specify --locked to avoid this.

can bump MSRV

**Comment by therealprof** [comment]
> can bump MSRV

I think that's probably the way to go...

**Comment by Emilgardis** [comment]
done, we should get a release out after this too

**Review by therealprof — DISMISSED** [review]
LGTM, thanks.

bors r+

**Comment by Emilgardis** [comment]
interesting.

bors retry

**Comment by Emilgardis** [comment]
bors cancel

the bors matrix is looking for version number

**Comment by Emilgardis** [comment]
bors try

should fail

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -69,3 +69,14 @@ jobs:
...
+    runs-on: ubuntu-latest
+    needs: ci
+    if: always()
+    steps:
+      - name: Result
+        run: |
+          jq -C <<< "${needs}"
+          "$(jq -r 'all(.result as $result | (["success"] | contains([$result])))' <<< "${needs}")"
+        env:
+          needs: ${{ toJson(needs) }}
```
Comment (Emilgardis) [inline .github/workflows/ci.yml:82]: technically, this is not needed. Could simply be

```suggestion
    steps:
      - name: Result
        run: exit 0
```

**Comment by Emilgardis** [comment]
bumping msrv due to https://github.com/rust-lang/rust/issues/105167 on macos

**Review by adamgreig — APPROVED** [review]
thanks, lgtm!

bors r+

**Pull request merged**